src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,113 @@
+in alphabetical order:
+
+[firebase3](https://pub.dartlang.org/packages/firebase3)
+: A Dart wrapper library for the Firebase.
+
+[intl](https://pub.dartlang.org/packages/intl)
+: Support for internationalization and localization facilities.
+
+[logging](https://pub.dartlang.org/packages/logging)
+: A configuratable mechanism for logging messages.
+
+[mockito](https://pub.dartlang.org/packages/mockito)
+: A Dart version of the Mockito framework.
+
+[path](https://pub.dartlang.org/packages/path)
+: A comprehensive, cross-platform path manipulation library.
+  See [Unboxing Packages: path](http://news.dartlang.org/2016/06/unboxing-packages-path.html)
+  for more information.
+
+[quiver](https://pub.dartlang.org/packages/quiver)
+: A set of utilitiy libraries that makes it easier and more convenient to use many Dart
+  libraries.
+
+[shelf](https://pub.dartlang.org/packages/shelf)
+: Web server middleware for Dart.
+
+[stagehand](https://pub.dartlang.org/packages/stagehand)
+: A Dart project generator.
+
+[test](https://pub.dartlang.org/packages/test)
+: A standard way of writing and running tests in Dart.
+
+Search for other available packages on [pub.dartlang.org](https://pub.dartlang.org/).
+
+## "Unboxing Packages" series
+
+Natalie Weizenbaum has written a series of posts on
+[news.dartlang.org](http://news.dartlang.org/) called "Unboxing Packages."
+Each post contains in-depth detail on how to use some of the Dart packages:
+
+* [Unboxing Packages: async, part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html)
+* [Unboxing Packages: async, part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html)
+* [Unboxing Packages: async, part 3](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html)
+* [Unboxing Packages: collection](http://news.dartlang.org/2016/01/unboxing-packages-collection.html)
+* [Unboxing Packages: json_rpc_2](http://news.dartlang.org/2016/05/unboxing-packages-jsonrpc2.html)
+* [Unboxing Packages: source_span](http://news.dartlang.org/2016/02/one-of-best-aspects-of-package.html)
+* [Unboxing Packages: stack_trace](http://news.dartlang.org/2016/01/unboxing-packages-stacktrace.html)
+* [Unboxing Packages: stream_channel](http://news.dartlang.org/2016/04/unboxing-packages-streamchannel.html)
+* [Unboxing Packages: string_scanner](http://news.dartlang.org/2016/02/unboxing-packages-stringscanner.html)
+* [Unboxing Packages: vm_service_client](http://news.dartlang.org/2016/05/unboxing-packages-vmserviceclient.html)
+
+## Importing libraries and packages
+
+If you need help with importing libraries and packages into your code, see
+[Using libraries](/guides/language/language-tour#libraries-and-visibility),
+a section in the [language tour](/guides/language/language-tour), and
+[Importing library files](/guides/libraries/create-library-packages#importing-library-files),
+a section in [Create Library Packages](/guides/libraries/create-library-packages).
```

Comment:
You should probably link to other sources of information about libraries.

For example, angular2 is one of the most popular libraries, but we don't cover it here because it's web-specific. (We should probably mention it, at least, as an example of a web-specific library.) Where should they go to find info about libraries that are useful for web apps? 

Server-side-only libraries should probably be similarly handled. And Flutter libs, too.